This adds missing imports from the main patch. Quarto-cli does not seem to care either way.
--- a/src/dev_import_map.json
+++ b/src/dev_import_map.json
@@ -53,7 +53,9 @@
     "cliffy/command/mod.ts": "./vendor/deno.land/x/cliffy@v0.25.7/command/mod.ts",
     "log/mod.ts": "./vendor/deno.land/std@0.177.0/log/mod.ts",
     "path/mod.ts": "./vendor/deno.land/std@0.177.0/path/mod.ts",
+    "streams/read_all.ts": "./vendor/deno.land/std@0.177.0/streams/read_all.ts",
     "streams/write_all.ts": "./vendor/deno.land/std@0.177.0/streams/write_all.ts",
+    "streams/copy.ts": "./vendor/deno.land/std@0.177.0/streams/copy.ts",
     "semver/mod.ts": "./vendor/deno.land/x/semver@v1.4.0/mod.ts",
     "node/fs.ts": "./vendor/deno.land/std@0.177.0/node/fs.ts",
     "encoding/yaml.ts": "./vendor/deno.land/std@0.177.0/encoding/yaml.ts",
@@ -112,6 +114,7 @@
     "encoding/_yaml/type/mod.ts": "./vendor/deno.land/std@0.177.0/encoding/_yaml/type/mod.ts",
     "media_types/mod.ts": "./vendor/deno.land/x/media_types@v2.10.1/mod.ts",
     "io/buffer.ts": "./vendor/deno.land/std@0.177.0/io/buffer.ts",
+    "io/read_lines.ts": "./vendor/deno.land/std@0.177.0/io/read_lines.ts",
     "xmlp/mod.ts": "./vendor/deno.land/x/xmlp@v0.2.8/mod.ts",
     "cliffy/prompt/input.ts": "./vendor/deno.land/x/cliffy@v0.25.7/prompt/input.ts",
     "crypto/mod.ts": "./vendor/deno.land/std@0.177.0/crypto/mod.ts",
--- a/src/vendor/import_map.json
+++ b/src/vendor/import_map.json
@@ -9,7 +9,9 @@
     "acorn/acorn": "./cdn.skypack.dev/acorn@7.4.1.js",
     "acorn/walk": "./cdn.skypack.dev/acorn-walk@7.2.0.js",
     "path/mod.ts": "./deno.land/std@0.177.0/path/mod.ts",
+    "streams/read_all.ts": "./deno.land/std@0.177.0/streams/read_all.ts",
     "streams/write_all.ts": "./deno.land/std@0.177.0/streams/write_all.ts",
+    "streams/copy.ts": "./deno.land/std@0.177.0/streams/copy.ts",
     "semver/mod.ts": "./deno.land/x/semver@v1.4.0/mod.ts",
     "node/fs.ts": "./deno.land/std@0.177.0/node/fs.ts",
     "encoding/yaml.ts": "./deno.land/std@0.177.0/encoding/yaml.ts",
@@ -72,6 +74,7 @@
     "observablehq/parser": "./cdn.skypack.dev/@observablehq/parser@4.5.0.js",
     "media_types/mod.ts": "./deno.land/x/media_types@v2.10.1/mod.ts",
     "io/buffer.ts": "./deno.land/std@0.177.0/io/buffer.ts",
+    "io/read_lines.ts": "./deno.land/std@0.177.0/io/read_lines.ts",
     "xmlp/mod.ts": "./deno.land/x/xmlp@v0.2.8/mod.ts",
     "cliffy/prompt/input.ts": "./deno.land/x/cliffy@v0.25.7/prompt/input.ts",
     "crypto/mod.ts": "./deno.land/std@0.177.0/crypto/mod.ts",
